### DISCUSSION TITLE
Flaw in Russian localization of Load Campaign menu

### DISCUSSION THREAD

**Discussion by perdakovich** [FakeFishGames/Barotrauma#13634] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

<details>

<summary> English current: </summary>

```xml
<campaignmode.incompatiblecontentpackage>The content package "[incompatiblecontentpackage]" was not in use when saving the campaign.</campaignmode.incompatiblecontentpackage>
<campaignmode.incompatiblecontentpackages>The content packages [incompatiblecontentpackages] were not in use when saving the campaign.</campaignmode.incompatiblecontentpackages>
```

</details>

<details>

<summary> Russian current: </summary>

```xml
<campaignmode.incompatiblecontentpackage>Набор контента "[incompatiblecontentpackage]" использовался при сохранении кампании.</campaignmode.incompatiblecontentpackage>
<campaignmode.incompatiblecontentpackages>Наборы контента "[incompatiblecontentpackages]" использовались при сохранении кампании.</campaignmode.incompatiblecontentpackages>
```

</details>

<details>

<summary> Russian proposal: </summary> 

```xml
<campaignmode.incompatiblecontentpackage>Набор контента "[incompatiblecontentpackage]" не использовался при сохранении кампании.</campaignmode.incompatiblecontentpackage>
<campaignmode.incompatiblecontentpackages>Наборы контента "[incompatiblecontentpackages]" не использовались при сохранении кампании.</campaignmode.incompatiblecontentpackages>
```

</details>

Потому что 'was not in use' == 'не использовался'

В нынешней локализации куда-то пропала 'не'. Очень путало меня, пока не посмотрел как оно в оригинале.

### Reproduction steps

*

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Linux

**Comment by Regalis11** [comment]
Thank you for the report! I've created [an issue ticket](https://github.com/FakeFishGames/Barotrauma/issues/13635) about this, we'll pass it to the localization team.
